package.json

```diff
@@ -143,13 +144,73 @@
                 "category": "Azure Resource Manager Tools",
                 "title": "Reload Cached Schemas",
                 "command": "azurerm-vscode-tools.reloadSchemas"
+            },
+            {
```

Comment:
Not sure it makes sense to expose multiple commands for this.  What scenario are you thinking of? Shortcut keys?

Reply:
If we decide to remove them, you could simply hide them from the command palette, e.g.:
![image](https://user-images.githubusercontent.com/6913354/74490157-83100d80-4e7c-11ea-96dd-5e85026052bb.png)

Reply:
> Not sure it makes sense to expose multiple commands for this. What scenario are you thinking of? Shortcut keys?

The reason is the context menu for the ARM Template Outline.
As a side benefit it was easy to write unit tests for the sorting.
I did not want them to be visible in the command palette so I will hide them in the way you described.

Reply:
I hid all commands except "Sort template"